src/Debugging/Drivers/ErrorLogDebugger.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Saloon\Debugging\Drivers;
+
+use Saloon\Debugging\DebugData;
+
+class ErrorLogDebugger extends DebuggingDriver
+{
+    public function name(): string
+    {
+        return 'error_log';
+    }
+
+    /**
+     * @param \Saloon\Debugging\DebugData $data
+     *
+     * @return void
+     * @throws \JsonException
+     */
+    public function send(DebugData $data): void
+    {
+        $encoded = json_encode($this->formatData($data), JSON_THROW_ON_ERROR);
+
+        error_log(print_r($encoded, true), LOG_DEBUG);
```

Comment:
`json_encode()` already turns it into a string, so no need for `print_r()`.

```suggestion
        error_log($encoded, LOG_DEBUG);
```